openmc/source.py

```diff
@@ -333,6 +406,212 @@ def from_xml_element(cls, elem: ET.Element, meshes=None) -> 'openmc.Source':
         return source
 
 
+class Source(IndependentSource):
+    """
+    A class for backward compatibility of sources. Will be remeved in the
+    future. Please update to IndependentSource.
+    """
+
+    def __init__(self, *args, **kwargs):
+        warnings.warn("This class is deprecated in favor of 'IndependentSource'", DeprecationWarning)
+        super().__init__(*args, **kwargs)
```

Comment:
So as not to introduce another class in the hierarchy:
```suggestion
def Source(*args, **kwargs):
    """
    A function for backward compatibility of sources. Will be removed in the
    future. Please update to IndependentSource.
    """
    warnings.warn("This class is deprecated in favor of 'IndependentSource'", FutureWarning)
    return openmc.IndependentSource(*args, **kwargs)
```

Reply:
Nice. I'll try to remember this pattern. 👍🏻